airflow/api_fastapi/core_api/openapi/v1-generated.yaml

```diff
@@ -5557,6 +5557,50 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/HTTPValidationError'
+  /public/parseDagFile/{file_token}:
+    put:
+      tags:
+      - DAG Parsing
+      summary: Reparse Dag File
+      description: Request re-parsing a DAG file.
+      operationId: reparse_dag_file
+      parameters:
+      - name: file_token
+        in: path
+        required: true
+        schema:
+          type: string
+          title: File Token
+      responses:
+        '200':
```

Comment:
Response code is not correct, this should be `201`

Reply:
Fixed this.